frontend/integration-tests-cypress/cypress/tests/monitoring.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { timeouts } from '../support/utils/timeouts';
+import { testName } from '../support/utils/common';
+import { submitButton, errorMessage } from '../support/utils/selectors';
+import { listPage } from '../support/views/list-page';
+import { detailsPage } from '../support/views/details-page';
+
+const isWatchdogAlertDetailsPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alert Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+const isWatchdogAlertRulesPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alerting Rule Details');
+  detailsPage.sectionHeaderShouldExist('Active Alerts');
+};
+
+const isWatchdogSilencePage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Silence Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+before(() => {
+  cy.createTestProject(testName);
+});
+after(() => {
+  cy.deleteTestProject(testName);
+});
+
+describe('Monitoring: Alerts', () => {
+  // each it() should be its own atomic test (run independently of other tests), per best practices.
+  it('displays and filters the Alerts list page, links to detail pages', () => {
+    cy.log('Use vert. nav. menu to goto Monitoring -> Alerting');
+    cy.clickNavLink(['Monitoring', 'Alerting']);
+    listPage.titleShouldBe('Alerting');
+    listPage.projectDropdownShouldNotExist();
+
+    cy.log('Filter Alerts');
+    listPage.filter.byName('Watchdog'); // needed 'pipe' for more specificity due to fuzzy search
+    listPage.rows.countShouldBe(1);
+    listPage.rows.shouldExist('Watchdog');
+
+    cy.log('Drills down to Alert details page');
+    listPage.rows.shouldExist('Watchdog').click();
+    isWatchdogAlertDetailsPage();
```

Comment:
Couple thoughts:

```
isWatchdogAlertDetailsPage()
// vs
shouldRedirectToWatchdogAlertDetailsPage()
shouldLoadWatchdogAlertDetailsPage()
```
I tend toward `should` as a consistent signal for assertion. 
There is also an implicit redirect in here, pondering if it's helpful to call that out.

Reply:
How about `shouldBeWatchdogAlertDetailsPage()' ?

Reply:
That does make me ask "what should be?".